service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -336,8 +332,11 @@ void shouldReturnSelectedHearings() {
             List<Element<HearingBooking>> hearingBookings = createHearingBookings();
             Selector selector = Selector.builder().selected(List.of(1)).build();
 
-            assertThat(service.getSelectedHearings(selector, hearingBookings).size()).isEqualTo(1);
-            assertThat(service.getSelectedHearings(selector, hearingBookings).get(0).getValue().getType())
+            List<Element<HearingBooking>> selectedHearings = service.getSelectedHearings(selector, hearingBookings);
```

Comment:
could be:
             assertThat(selectedHearings)
                .extracting(hearing -> hearing.getValue().getType())
                .containsExactly(CASE_MANAGEMENT);